keras_cv/models/object_detection/yolo_v8/yolo_v8_detector_test.py

```diff
@@ -149,9 +150,14 @@ def test_saved_model(self, save_format, filename):
         )
         xs, _ = _create_bounding_box_dataset("xywh")
         model_output = model(xs)
-        save_path = os.path.join(self.get_temp_dir(), filename)
-        model.save(save_path, save_format=save_format)
-        restored_model = keras.models.load_model(save_path)
+        save_path = os.path.join(
+            self.get_temp_dir(), "yolo_v8_xs_detector.keras"
+        )
+        model.save(save_path)
+        restored_model = keras.saving.load_model(
+            save_path,
+            custom_objects={"YOLOV8Detector": keras_cv.models.YOLOV8Detector},
```

Comment:
Let's add a TODO to figure out how to remove this in the future

Reply:
Done, thanks!